crypto/evp_extra/p_ed25519_asn1.c

```diff
@@ -281,3 +281,28 @@ const EVP_PKEY_ASN1_METHOD ed25519_asn1_meth = {
     NULL /* param_cmp */,
     ed25519_free,
 };
+
+const EVP_PKEY_ASN1_METHOD ed25519ph_asn1_meth = {
```

Comment:
Is this a place holder for differentiating changes that may come to ed25519ph or there is something we need from this method in this PR that is different from that of ed25519?

Reply:
This is needed even though we don't support the ASN.1 PKCS#8 or SubjectPublicKeyInfo format serialization/deserialization with Ed25519ph. This method table is required though for the functionality of setting the raw public key or private keys on an EVP structure. Thus is why it is here.